### ISSUE TITLE
Possible overflow in mmsMsg_createExtendedFilename()

### ISSUE THREAD

**Issue by stv0g** [mz-automation/libiec61850#310]
```
In function 'mmsMsg_createExtendedFilename',
    inlined from 'mmsMsg_openFile' at /tmp/tmp.heDP51Tfuh/libiec61850/src/mms/iso_mms/common/mms_common_msg.c:472:5:
/tmp/tmp.heDP51Tfuh/libiec61850/src/mms/iso_mms/common/mms_common_msg.c:455:5: warning: 'strncat' specified bound 512 equals destination size [-Wstringop-overflow=]
     strncat(extendedFileName, fileName, 512);
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by mzillgith** [comment]
Thanks. It seems already fixed in v1.5. I will also fix it in v1.4.

**Comment by stv0g** [comment]
Thanks @mzillgith :)
